### ISSUE TITLE
replay pipeline records not show

### ISSUE THREAD

**Issue by Aaron199** [kubesphere/ks-devops#772]
**Describe the Bug**
When executing the pipeline through replay, it cannot be shown in the record list and is already running in the Jenkins console in real time
ks console:
![WX20220803-172706](https://user-images.githubusercontent.com/35388612/182646567-a84ac27c-3b86-490c-82a5-0b40af7b3508.png)

jenkins console：
![WX20220803-172736](https://user-images.githubusercontent.com/35388612/182646591-d34b301c-1b5d-468e-b274-dc387bd692d6.png)

i found "PipelineRun" only three records .
![WX20220803-173023](https://user-images.githubusercontent.com/35388612/182648531-5f2ed129-0647-4394-a781-049ba7b39e70.png)

**Versions Used**
KubeSphere:  from 3.1.0 upgrade 3.3.0
Kubernetes: (If KubeSphere installer used, you can skip this)
v1.19.10

**Environment**
How many nodes and their hardware configuration:

For example: CentOS 7.5 / 3 masters:  8cpu/8g; 3 nodes: 8cpu/16g
(and other info are welcomed to help us debugging)
Ubuntu 18.04.5  8C32G

**How To Reproduce**

**Expected behavior**

**Comment by Aaron199** [comment]
can anyone help me?

**Comment by HChef** [comment]
i have same question!

**Comment by LinuxSuRen** [comment]
Please add a webhook address to the following ConfigMap to fix it if you are using 3.3+

![image](https://user-images.githubusercontent.com/1450685/187603989-b2bfbc0f-65eb-4239-94ea-049b8d7e96b1.png)

```yaml
unclassified:
  eventDispatcher:
    receiver: http://devops-apiserver.kubesphere-devops-system:9090/v1alpha3/webhooks/jenkins
```

**Comment by Aaron199** [comment]
it worked, thx
